pom.xml

```diff
@@ -2,8 +2,8 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.bitbucket.ucchy</groupId>
   <artifactId>UndineMailer</artifactId>
-  <version>1.10.3</version>
-  <description></description>
+  <version>1.11</version>

```

Comment:
I'd avoid doing version changes in PRs as there may be other changes included in a version bump.

Reply:
Yes, sir.
Am I correct in thinking that this is something that the owner specifies when there is a Release?

Reply:
yes

Reply:
fe30a24172a789760b991cfde84f33b458ce5db4 Fixed,
Thank you for reviewing